app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,84 @@
+class ReviewsController < ApplicationController
+  before_action :set_review, only: [:show, :edit, :update, :destroy]
+  before_filter :load_resturant
```

Comment:
I probably used an old tutorial in order to make those changes... fixed it now :)